assets/js/modules/adsense/index.js

```diff
@@ -75,20 +75,17 @@ export const registerModule = ( modules ) => {
 			__( 'AdSense insights through Site Kit', 'google-site-kit' ),
 		],
 		checkRequirements: async ( registry ) => {
-			const adBlockerActive = await registry
+			const adBlockerActiveMsg = await registry
 				.__experimentalResolveSelect( MODULES_ADSENSE )
-				.isAdBlockerActive();
+				.getAdBlockerWarningMessage();
```

Comment:
This is the reason VRT and E2E tests were failing; `resolveSelect` will return a value after the selector's resolver has completed, but because the IB suggested changing the selector used, this returns immediately because `getAdBlockerWarningMessage` doesn't _have_ an associated selector.

There are two options here:

1. continue using `isAdBlockerActive` here but getting the message later on if `isAdBlockerActive`  is true
2. add a (redundant) resolver to `getAdBlockerWarningMessage`

I added a commit that does the former. It's not 100% in line with the IB but still fulfills the ACs and behaves as-intended.